src/scripts/Save.ts

```diff
@@ -23,12 +23,8 @@ class Save {
         let element = document.createElement('a');
         element.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(btoa(JSON.stringify(player))));
         let currentdate = new Date();
-        let datetime = "" + currentdate.getDate() + "/"
-            + (currentdate.getMonth() + 1) + "/"
-            + currentdate.getFullYear() + " @ "
-            + currentdate.getHours() + ":"
-            + currentdate.getMinutes();
-        let filename = "Pokeclicker save - " + datetime + '.txt';
+        let datestr = currentdate.toISOString().replace("T", " ").slice(0, 19);
```

Comment:
This will still leave you with: `2019-05-30 01:48:25` which _should_ show as `2019-05-30 01_48_25` on windows.
I feel like it would be best to have a format that will show the same on all systems?

Reply:
I think as long as it's consistent within one's own system, we're good. The goal is for players to be able to sort the files and find the one they want easily.